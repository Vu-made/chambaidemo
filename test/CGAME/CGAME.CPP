#include <bits/stdc++.h>
#define N 100005
using namespace std;
int a[N];
int s1 = 0, s2 = 0, n;
int main()
{
    freopen("CGAME.INP","r",stdin);
    freopen("CGAME.OUT","w",stdout);
    cin >> n;
    for (int i=1; i<=n; i++) cin >> a[i];
    int i = 1, j = n, d = 0;
    while (i <= j)
    {
        if (d == 0) s1 += max(a[i],a[j]);
        else s2 += max(a[i],a[j]);

        if (a[i] > a[j]) i++;
        else j--;
        d = 1 - d;
    }
    cout << s1 << " " << s2;
    return 0;
}
