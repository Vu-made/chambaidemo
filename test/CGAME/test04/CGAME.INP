79
53 44 2 14 64 20 28 24 9 29 67 17 37 40 56 27 52 46 3 70 62 11 30 45 49 16 71 15 39 51 77 6 47 50 31 36 75 38 10 21 32 66 72 35 12 13 78 68 69 63 74 57 33 65 23 60 34 76 54 55 48 4 59 58 25 19 18 43 8 7 1 26 73 41 22 42 61 5 79 