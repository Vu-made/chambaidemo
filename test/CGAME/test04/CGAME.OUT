1728 1432